examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue

```diff
@@ -65,15 +58,12 @@ export default {
     getContent({
       model: 'page',
       apiKey: BUILDER_PUBLIC_API_KEY,
-      options: getBuilderSearchParams(
-        convertSearchParamsToQueryObject(new URLSearchParams(window.location.search))
-      ),
       userAttributes: {
         urlPath: window.location.pathname,
       },
     }).then(res => {
       this.content = res;
-      this.canShowContent = this.content || isEditing() || isPreviewing();
+      this.canShowContent = this.content || isPreviewing();
```

Comment:
See this file for how the builder integration is simplified for all SDK examples

Reply:
cc @steve8708 this is how examples look after the changes we talked about

Reply:
😍